packages/dara-core/js/components/side-bar-frame/side-bar-frame.tsx

```diff
@@ -107,9 +108,9 @@ const LogoutArrow = getIcon('fa-solid fa-arrow-right-from-bracket');
 function SideBarFrame(props: SideBarFrameProps): JSX.Element {
     const theme = useTheme();
     const { data: config } = useConfig();
-    const logo = props.logo_path && <LogoImage alt="Logo" src={props.logo_path} width={props.logo_width} />;
-    const daraLogo =
-        theme.themeType === 'dark' ? <img alt="Dara" src={DaraDark} /> : <img alt="Dara" src={DaraLight} />;
+    const logo = props.logo_path && <LogoImage alt="Logo" src={prependBaseUrl(props.logo_path)} width={props.logo_width} />;
+    const logoSrc = theme.themeType === 'dark' ? DaraDark : DaraLight;
+    const daraLogo = <img alt="Dara" src={prependBaseUrl(logoSrc)} />
```

Comment:
There's two logos here:
- the Dara logo (Dark or Light) does not need prepending as it's imported directly as an asset
- the `props.logo_path` logo is the one that does need prepending

So you can remove the prepend from logoSrc

Reply:
Removed prepending from `daraLogo` in both cases.